### ISSUE TITLE
Add ability to filter pokemon available in a region, not just when they're intoduced

### ISSUE THREAD

**Issue by Kyrushi** [pokeclicker/pokeclicker#2320]
Currently when you organize by region, it only shows you what pokemon were INTRODUCED in said region, not which pokemon are available in the region as a whole. This is annoying when using the standard ruleset which includes Regional Attack Debuff because i'm purposely trying to breed only what will give me an advantage in the current region. 

I suggest either changing the existing region filter to include older pokemon available in the selected region or to add a new type of regional filter. As an example of what i'd like to see: If I select Sinnoh, it ought to show older pokemon that originated from other regions, but are available for capture in Sinnoh (e.g. Geodude, Misdreavus, Mageneton.)

tl;dr if the pokemon would have their full base attack in the region you selected in the filter, it should be displayed. Not just the pokemon introduced in said region.

**Comment by Ultima1990** [comment]
Native = the mons that are introduced in the region. Geodude is not native in Sinnoh. So in the standard ruleset with regional attack debuff filtering by region does exactly what you want.

**Comment by Kyrushi** [comment]
Well, I've been wasting a lot of time then. I simply misunderstood what the game meant by native considering that native usually means that a creature is indigenous to an area. I'd suggest instead changing the description to something like "pokemon introduced in" but I'm gonna assume this is specifically a me problem. Thank you for your time.
